NuGet.Core.Config

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Just to give a "standard" nuget configuration for use when build/testing on core. Core builds require a different nuget config to msbuild builds/tests.